site/en/tutorials/quickstart/beginner.ipynb

```diff
@@ -116,6 +117,8 @@
         "id": "7NAbSZiaoJ4z"
       },
       "source": [
+        "**Note:** If you are running in a local environment, make sure you have upgraded to the latest `pip` to install the TensorFlow 2 package. See the [install guide](https://www.tensorflow.org/install) for details.\n",
```

Comment:
Maybe we could be more specific here. Like:

"""
Note: If `pip install tensorflow` fails with an error like `ERROR: No matching distribution found for tensorflow` Upgrade your pip installation and check the python version requireents in the [install guide](...)
"""

Since you get that error if the pip version or python version is out of range.